css/about.css

```diff
@@ -0,0 +1,56 @@
+/* About Section Starts Here */
+.about {
+  background-color: #fff;
+  border-radius: 0 8em 0 0;
+  margin: 8em 0 0 0;
+}
+
+.headline-skill-container {
+  padding: 5em 7% 0 7%;
+}
+
+.about-heading-small {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.about-skills-container {
+  display: flex;
+  flex-direction: column;
+  padding: 2em 1em;
+  width: 100%;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/60083437/235314066-b41a1169-c676-4a08-9978-db4b896a03c4.png)
- [ ] I have marked `4` problems for the `about` section. Kindly fix those issues to match the given design guideline.
- Problem No. `1` and `2` are related to margins/gaps/padding. There are to much space between the contents.
- Problem No. `3` and `4` are related to `underline`. According to the given design there should be two underline after `Frameworks` and `Skills`.